tests/agents/json-specs/cgroup_parsing.json

```diff
@@ -0,0 +1,22 @@
+{
+	"testUnderscores": {
+		"groupLine": "1:name=systemd:/kubepods.slice/kubepods-burstable.slice/kubepods-burstable-pod90d81341_92de_11e7_8cf2_507b9d4141fa.slice/crio-2227daf62df6694645fee5df53c1f91271546a9560e8600a525690ae252b7f63.scope",
+		"containerId": "2227daf62df6694645fee5df53c1f91271546a9560e8600a525690ae252b7f63",
+		"podId": "90d81341-92de-11e7-8cf2-507b9d4141fa"
+	},
+	"testOpenshiftForm": {
+		"groupLine": "9:freezer:/kubepods.slice/kubepods-pod22949dce_fd8b_11ea_8ede_98f2b32c645c.slice/docker-b15a5bdedd2e7645c3be271364324321b908314e4c77857bbfd32a041148c07f.scope",
+		"containerId": "b15a5bdedd2e7645c3be271364324321b908314e4c77857bbfd32a041148c07f",
+		"podId": "22949dce-fd8b-11ea-8ede-98f2b32c645c"
+	},
+	"testUbuntuCGroup": {
```

Comment:
Is it possible to link from textual documentation to this file as an attempt to make sure we don't forget about it ? It might also be a good idea to provide a bit more context about the JSON spec content, for example which file is being parsed in the filesystem.

Reply:
Good point, I'll add the link to this test spec from https://github.com/elastic/apm/blob/main/specs/agents/metadata.md.
As for the context, I think this file already does a good job of describing that - WDYT?

Reply:
Yes, this definitely does a good job of describing the context, so in fact just a link to this JSON spec might be enough.